aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -156,6 +156,21 @@ Tensor& addr_out(Tensor &result, const Tensor& self, const Tensor& vec1, const T
   return at::_addr_out(result, self, vec1, vec2, beta, alpha);
 }
 
+Tensor& ger_out(Tensor &result, const Tensor& self, const Tensor& vec2) {
+  check_1d(self, "self", "ger");
+  check_1d(vec2, "vec2", "ger");
+  if (result.numel() == 0) {
```

Comment:
This change is BC breaking --
```
x = torch.randn(5)
y = torch.randn(7)
out = torch.randn(2, 3)
torch.ger(x, y, out=out) # used to be OK.
```

If we want to change the semantics we should do a deprecation cycle by allowing the old behavior (resize anything to the correct size) but raise a deprecation warning.

Reply:
Ah, I see. I will support the old behavior for now but print out a warning letting user switch to this new convention.

Reply:
back to old behavior as we discussed